cfg/config.go

```diff
@@ -750,6 +750,12 @@ func BuildFlagSet(flagSet *pflag.FlagSet) error {
 		return err
 	}
 
+	flagSet.BoolP("enable-http-dns-cache", "", false, "Enables DNS cache for HTTP/1 connections")
```

Comment:
so it will be false by default and each users need to set it to opt-in?

Reply:
No, we will set it to true in a subsequent pr.